build.sbt

```diff
@@ -151,6 +151,24 @@ lazy val repl = project
 //    fork in (Test, testOnly) := true
   )
 
+
+/**
+ * REPL available via remote ssh access. Pluggable into any app environment to allow live hacking on a live application.
+ */
+lazy val sshd = project.
+    dependsOn(repl).
+    settings(sharedSettings:_*).
+    settings(
+      name := "ammonite-repl-sshd",
+      libraryDependencies ++= Seq(
+        "org.apache.sshd" % "sshd-core" % "0.14.0",
+        //-- test --//
+        "org.slf4j" % "slf4j-nop" % "1.7.12" % "test",
+        "com.jcraft" % "jsch" % "0.1.53" % "test",
+        "org.scalacheck" %% "scalacheck" % "1.12.4" % "test"
```

Comment:
I don't really see the value of ScalaCheck in this test suite. Do we think that "some, but not all, incorrect credentials will pass" is a likely failure mode? It doesn't seem likely to me, but maybe you have some failure modes in mind I haven't thought about